src/app/FormCreate.jsx

```diff
@@ -1,21 +1,28 @@
+
+/**
+ * Module dependencies
+ */
+
 import React, { Component } from 'react';
 import Radium from 'radium';
 import { connect } from 'react-redux';
 
 import { createEmpty, updateForm } from 'forms/FormActions';
 import { showFlashMessage } from 'flashmessages/FlashMessagesActions';
-
 import Page from 'app/layout/Page';
 import FormBuilder from 'forms/FormBuilder.js';
 import FormChrome from 'app/layout/FormChrome';
 
-@connect(({ app, forms }) => ({ app, forms }))
+@connect(({ app, forms }) => ({
```

Comment:
I love using connectors for Elkhorn.  This looks like a graet pattern to include as we address plugins and start creating some boilerplate for accessing our services.